### DISCUSSION TITLE
How do I add an action rule on header button?

### DISCUSSION THREAD

**Discussion by planthom** [Power-Components/livewire-powergrid#895] [Q&A]
So I have a bulk delete button on header.
However when I try to put action rules on the button, nothing happen.
It works if I use it on row button, but that's not what I want.

The rule is, whenever a user checked more than two rows, a bulk delete button is appear. so the default is hidden.

```
public function header(): array
    {
        return [
            Button::make('delete-selected', 'Delete Checked Rows')
            ->class('bg-red-500 text-white px-2 py-2 rounded-lg')
            ->emit('bulkDeleteActionEvent', [])
        ];
    }


public function actionRules(): array
    {
       return [
            Rule::button('delete-selected')
                ->when(fn($model) => count($this->checkboxValues) < 2)
                ->hide(),
        ];
    }
```

**Comment by planthom** [comment]
Hi, is it possible or not? I really need insight from you guys

**Comment by luanfreitasdev** [comment]
At the moment it is not possible. I must think of something about it
